libs/core/src/lib.rs

```diff
@@ -372,6 +372,14 @@ impl<Client: Requester> CoreLib<Client> {
         })
     }
 
+    #[instrument(level = "debug", skip(self), err(Debug))]
+    pub fn get_most_active_documents(&self) -> Result<Vec<(Uuid, usize)>, Error<GetUsageError>> {
```

Comment:
Hey, @SmailBarkouch, I exposed an endpoint in core `get_most_active_documents ` that you can use in android to improve the "Recent Documents" view. This endpoint will recommend more relevant files for the user.

The current approach for recommendation is to build a tree of all the files the user has access to, then sort them based on the newest. 
This new approach measures the count of read/write operations per document, then returns the most active documents (the ones with the most reads/writes).

This alternative approach will thus give more engaging file suggestions. How does this sound to you? Any editions or modifications you recommend?

Reply:
I think this is a good approach, but I would somewhat argue there should be a balance between ranking what is accessed most, and what was most recently used. I feel this approach kinda makes "Recent Documents" into "Most Accessed Documents", as the most recently used file may not be accessed nearly as much (like a new file).

How this balance should be achieved is arguable. One approach I am thinking of is first ordering documents into "buckets" of time ranges (within 5 hour, 5 hour to 1 day, 1 day to 3 days, and so on), and within those ranges you do these read and write comparisons. To be honest this does complicate it a lot so definitely some other approach may be preferred.